org.alloytools.alloy.core/src/main/java/edu/mit/csail/sdg/translator/TranslateAlloyToKodkod.java

```diff
@@ -69,13 +69,17 @@
 import kodkod.ast.operator.ExprOperator;
 import kodkod.engine.CapacityExceededException;
 import kodkod.engine.fol2sat.HigherOrderDeclException;
+import kodkod.engine.ltl2fol.TemporalTranslator;
 import kodkod.instance.Tuple;
 import kodkod.instance.TupleFactory;
 import kodkod.instance.TupleSet;
 import kodkod.util.ints.IntVector;
 
 /**
  * Translate an Alloy AST into Kodkod AST then attempt to solve it using Kodkod.
+ *
+ * @modified: Nuno Macedo, Eduardo Pessoa // [HASLab] electrum-temporal,
+ *            electrum-unbounded, electrum-decomposed
```

Comment:
adds the translation of temporal operators and quantifies globally over time constraints over sigs and fields (sig facts are implicitly globally quantified)